src/style.css

```
@@ -1 +1,120 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
```

do not repeat ypur self
```suggestion
  justify-content: center;
  margin: 20px auto 0;
  width: 100%;
}

.search-form_big {
  height: 70px;
}

.search-form_small {
  height: 42px;
}
```